zest/releaser/release.py

```diff
@@ -121,10 +121,14 @@ def _upload_distributions(self, package):
             result = utils.execute_command(utils.setup_py('bdist_wheel'))
         utils.show_interesting_lines(result)
         if not self.pypiconfig.is_pypi_configured():
-            logger.warn("You must have a properly configured %s file in "
-                        "your home dir to upload to a package index.",
-                        pypi.DIST_CONFIG_FILE)
-            return
+            logger.error(
+                "You must have a properly configured %s file in "
+                "your home dir to upload to a package index.",
```

Comment:
"... to a PYTHON package index."? To make it clearer that it is for python? 

Though.... I assume this code won't get run for non-python packages, right?